centreon/www/class/centreonMonitoring.class.php

```diff
@@ -118,63 +118,76 @@ public function getServiceStatusCount($host_name, $objXMLBG, $o, $status, $obj)
     }
 
     /**
-     *
-     * Proxy function
-     * @param unknown_type $hostList
-     * @param unknown_type $objXMLBG
-     * @param unknown_type $o
-     * @param unknown_type $instance
-     * @param unknown_type $hostgroups
+     * @param string $hostList
+     * @param CentreonXMLBGRequest $objXMLBG
+     * @param string $o
+     * @param false|int $instance
+     * @param false|int $hostgroups
      */
     public function getServiceStatus($hostList, $objXMLBG, $o, $instance, $hostgroups)
     {
-        if ($hostList == "") {
-            return array();
+        if ($hostList === '') {
+            return [];
         }
 
-        $rq = "SELECT h.name, s.description as service_name, s.state, s.service_id, "
-            . " (case s.state when 0 then 3 when 2 then 0 when 3 then 2  when 3 then 2 else s.state END) as tri "
-            . "FROM hosts h, services s ";
+        $rq = <<<SQL
```

Comment:
Just a small non-blocking comment, but as there is no variable to resolve in the request I guess you could use the Nowdoc’s syntax
```suggestion
        $rq = <<<'SQL'
```